### PR TITLE
Assorted missing docs for RC1

### PR DESCRIPTION
**Pull request by carlossanlop**

### CODE REVIEW

File: xml/System.Diagnostics/TagList.xml [hunk 1]
```diff
@@ -426,7 +426,8 @@
       </ReturnValue>
       <Docs>
         <summary>Gets a value indicating whether the <see cref="T:System.Diagnostics.TagList" /> is read-only. This property will always return <see langword="false" />.</summary>
-        <value>To be added.</value>
+        <value>
+          <see langword="true" /> if the <see cref="T:System.Collections.Generic.ICollection`1" /> is read-only; otherwise, <see langword="false" />.</value>
```
Comment (gewarren) [inline xml/System.Diagnostics/TagList.xml:430]: This contradicts with the summary, which says it always returns false.

File: xml/System.Numerics/BigInteger.xml [hunk 2]
```diff
@@ -1528,8 +1528,8 @@
         <Parameter Name="value" Type="TOther" Index="0" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <typeparam name="TOther">To be added.</typeparam>
-        <param name="value">To be added.</param>
+        <typeparam name="TOther">The type of <paramref name="value" />.</typeparam>
+        <param name="value">The value which is used to create the instance of <typeparamref name="TSelf" />.</param>
```
Comment (gewarren) [inline xml/System.Numerics/BigInteger.xml:1532]:
```suggestion
        <param name="value">The value that's used to create the instance of <typeparamref name="TSelf" />.</param>
```

File: xml/System.Numerics/BigInteger.xml [hunk 3]
```diff
@@ -1571,8 +1571,8 @@
...
```
Comment (gewarren) [inline xml/System.Numerics/BigInteger.xml:1575]:
```suggestion
        <param name="value">The value that's used to create the instance of <typeparamref name="TSelf" />.</param>
```

File: xml/System.Numerics/BigInteger.xml [hunk 4]
```diff
@@ -1685,8 +1685,8 @@
         <Parameter Name="right" Type="System.Numerics.BigInteger" Index="1" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
+        <param name="left">The value which <paramref name="right" /> divides.</param>
+        <param name="right">The value which divides <paramref name="left" />.</param>
```
Comment (gewarren) [inline xml/System.Numerics/BigInteger.xml:1689]:
```suggestion
        <param name="right">The value that divides <paramref name="left" />.</param>
```

File: xml/System.Numerics/BigInteger.xml [hunk 5]
```diff
@@ -1685,8 +1685,8 @@
         <Parameter Name="right" Type="System.Numerics.BigInteger" Index="1" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
+        <param name="left">The value which <paramref name="right" /> divides.</param>
```
Comment (gewarren) [inline xml/System.Numerics/BigInteger.xml:1688]:
```suggestion
        <param name="left">The value that <paramref name="right" /> divides.</param>
```

File: xml/System.Numerics/BigInteger.xml [hunk 6]
```diff
@@ -8979,8 +8979,8 @@ If `provider` is `null`, the <xref:System.Globalization.NumberFormatInfo> object
         <Parameter Name="rotateAmount" Type="System.Int32" Index="1" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="rotateAmount">To be added.</param>
+        <param name="value">The value which is rotated left by <paramref name="rotateAmount" />.</param>
```
Comment (gewarren) [inline xml/System.Numerics/BigInteger.xml:8982]:
```suggestion
        <param name="value">The value that's rotated left by <paramref name="rotateAmount" />.</param>
```

File: xml/System.Numerics/BigInteger.xml [hunk 7]
```diff
@@ -9016,8 +9016,8 @@ If `provider` is `null`, the <xref:System.Globalization.NumberFormatInfo> object
         <Parameter Name="rotateAmount" Type="System.Int32" Index="1" FrameworkAlternate="net-7.0" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="rotateAmount">To be added.</param>
+        <param name="value">The value which is rotated right by <paramref name="rotateAmount" />.</param>
```
Comment (gewarren) [inline xml/System.Numerics/BigInteger.xml:9019]:
```suggestion
        <param name="value">The value that's rotated right by <paramref name="rotateAmount" />.</param>
```

**Review by gewarren — APPROVED** [review]

File: xml/System.Diagnostics/TagList.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.Diagnostics/TagList.xml:430]: @tarekgh

**Review by rzikm — APPROVED** [review]
System.Net.* LGTM

File: xml/System.Diagnostics/TagList.xml (hunk 1 above)
Comment (tarekgh) [reply xml/System.Diagnostics/TagList.xml:430]: @gewarren is correct. We always return false here.

**Comment by tarekgh** [comment]
Other than @gewarren comment https://github.com/dotnet/dotnet-api-docs/pull/8391#pullrequestreview-1108352160, `System.Diagnostics` LGTM.

File: xml/System.Diagnostics/TagList.xml (hunk 1 above)
Comment (carlossanlop) [reply xml/System.Diagnostics/TagList.xml:430]:
```suggestion
          Always returns <see langword="false" />.</value>
```

**Pull request merged**